src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Internal/MsQuicApi.cs

```diff
@@ -38,7 +36,7 @@ private MsQuicApi(QUIC_API_TABLE* apiTable)
         {
             ApiTable = apiTable;
 
-            fixed (byte* pAppName = s_appName)
+            fixed (byte* pAppName = "System.Net.Quic"u8)
```

Comment:
I don't see the length being passed down to p/invoke, so it's likely the callee expects _AppName_ to be null-terminated. Since arrays are not guaranteed null-terminated, this likely only ever worked by happy accident before.

Since _u8_ literals are now guaranteed null-terminated this isn't a problem going forward. But I thought it was an interesting observation nonetheless.

Reply:
I sent this example off to the folks working on the analyzer.